tools/common/Optimizations.cs

```diff
@@ -249,6 +256,9 @@ public void Initialize (Application app)
 				RemoveUnsupportedILForBitcode = app.Platform == Utils.ApplePlatform.WatchOS && app.IsDeviceBuild;
 			}
 #endif
+			// Always enable Runtime.CpuArchitecture inlining.
```

Comment:
That's more enabling _by default_ than _always_ enabled :)

Reply:
I've updated the comment.